common/src/main/java/net/xolt/freecam/tripod/TripodSlot.java

```diff
@@ -0,0 +1,35 @@
+package net.xolt.freecam.tripod;
+
+import static org.lwjgl.glfw.GLFW.GLFW_KEY_0;
+
+public enum TripodSlot {
+    NONE, ONE, TWO, THREE, FOUR, FIVE, SIX, SEVEN, EIGHT, NINE;
+
+    public static final int MIN = 1;
+    public static final int MAX = values().length - 1;
+
+    @Override
+    public String toString() {
+        return this == NONE ? "None" : String.valueOf(ordinal());
```

Comment:
```suggestion
        return this == NONE ? "None" : "#%d".formatted(ordinal());
```

Could move the `#` here?